src/main/java/Ava.java

```diff
@@ -0,0 +1,107 @@
+                continue;
+            } else if (task.equals("list")) {
+                listTask(tasks);
+                continue;
+            } else if (task.contains("mark")) {
+                markTask(tasks, task);
+                continue;
+            } else if (task.startsWith("todo")) {
+                task = task.replace("todo ", "");
+                tasks[Task.getNumberOfTasks()] = new ToDo(task);
+            } else if (task.startsWith("deadline")) {
+                String[] taskAndDate = trimTask(task, "deadline ");
+                tasks[Task.getNumberOfTasks()] = new Deadline(taskAndDate[0], taskAndDate[1]);
+            } else if (task.startsWith("event")) {
+                String[] taskAndDate = trimTask(task, "event ");
+                tasks[Task.getNumberOfTasks()] = new Event(taskAndDate[0], taskAndDate[1], taskAndDate[2]);
+            }
+            addTask(tasks);
+        }
+    }
+
+    public static String[] trimTask(String task, String type) {
+        task = task.replace(type, "");
+        return task.split("/");
+    }
+
+    public static void addTask(Task[] tasks) {
+        printLine();
+        System.out.println("Got it! I've added this task:");
+        System.out.println(tasks[Task.getNumberOfTasks() - 1].toString());
+        if (Task.getNumberOfTasks() == 1) {
+            System.out.println("Now you have " + 1 + " task in the list~~~");
+        } else {
+            System.out.println("Now you have " + Task.getNumberOfTasks() + " tasks in the list~~~");
+        }
+        printLine();
+    }
+
+    public static void markTask(Task[] tasks, String command) {
+        printLine();
+        boolean isMark = true;
+        int taskChanged = 0;
+        if (command.startsWith("unmark")) {
+            isMark = false;
+        }
+        if (isMark) {
+            command = command.replace("mark ", "");
+            taskChanged = Integer.parseInt(command) - 1;
+            tasks[taskChanged].markAsDone();
+            System.out.println("Nice! I've marked this task as done:");
+        } else {
+            command = command.replace("unmark ", "");
+            taskChanged = Integer.parseInt(command) - 1;
+            tasks[taskChanged].markAsNotDone();
+            System.out.println("OK, I've marked this task as not done yet:");
+        }
+        System.out.println(tasks[taskChanged].toString());
+        printLine();
+    }
+
+    public static void listTask(Task[] tasks) {
+        printLine();
+        System.out.println("Here are the tasks in your list:");
+        int noOfTask = 0;
+        while (noOfTask < Task.getNumberOfTasks()) {
+            System.out.println((noOfTask + 1) + "." + tasks[noOfTask].toString());
+            noOfTask += 1;
+        }
+        printLine();
+    }
+
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void greet() {
+        printLine();
```

Comment:
Maybe you could use "\n" to create a newline instead of habing to call printLine()?